### DISCUSSION TITLE
my page that I deployed to GitHub Pages is giving a 404 error

### DISCUSSION THREAD

**Discussion by Vineeeee** [community/community#142183] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

https://github.com/Vineeeee/angularshop.github.io

I'm trying to deploy using Jekyll and GitHub Pages, but I'm only getting a 404 error: 
![image](https://github.com/user-attachments/assets/a97603ac-d238-4c2e-aac7-afc35af02498)

**Comment by ELHart05** [comment]
In general Github pages are not meant to host this kind of applications as it's meant for simple html, css, js files or in general for static websites, for this project you can use [vercel](https://vercel.com) or [netlify](https://www.netlify.com/) or even [render](https://render.com/).
